miio/gateway.py

```diff
@@ -847,9 +872,25 @@ def unpair(self):
     @command()
     def get_battery(self):
         """Update the battery level and return the new value."""
-        self._battery = self.send("get_battery").pop()
+        if self._gw.model != "lumi.gateway.mieu01":
```

Comment:
The docstring seems to be incorrect, I think it should be something like "Return the battery level, if available". It is also better not to check against the model number (or at least create a constant for that), but try to fetch the state and return None (and log a warning) if it fails. Or that's my opinion :-)

Reply:
@rytilahti I have updated the docstrings and made constants for the gateway models I know.

I would like to check this against the gateway_model because the battery level and voltage are used in the `__repr__` function, since the model is stored you only need 1x communication with the gateway to get the model but for all subsequent subdevices the model is already stored in memory such that you don't have aditional communication with the gateway while we already know that it is going to fail...

Could also make a dedicated variable like _supports_battery that will be set to falls after the first try fails. In that way we can catch the error withouth checking for the gateway_model and still limit the gateway communication to 1x.

What do you prefer?